tests/testthat/test-strategy-ff16w.R

```diff
@@ -17,19 +17,23 @@ test_that("FF16w Environment", {
   expect_equal(e$soil$rates, 0)
   
   # Make it rain
-  e$set_soil_infiltration_rate(10)
+  x <- seq(0, 9, 1)
+  y <- rep(1, 10)
+  e$rainfall_init(x, y)
   expect_equal(e$soil$states, 0)
   e$compute_rates()
-  expect_equal(e$soil$rates, 10)
+  expect_equal(e$soil$rates, 1)
   
+  # not needed anymore? should we still be able to set the states manually?
```

Comment:
probably. We will occasionally want to specify an exact environment for a non-dynamic use